test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1,9 +1,10 @@
 // RUN: circt-opt -pass-pipeline='builtin.module(firrtl.circuit(firrtl-lower-types))' --allow-unregistered-dialect %s | FileCheck --check-prefixes=CHECK,COMMON %s
 // RUN: circt-opt -pass-pipeline='builtin.module(firrtl.circuit(firrtl-lower-types{preserve-aggregate=all}))' --allow-unregistered-dialect %s | FileCheck --check-prefixes=AGGREGATE,COMMON %s
-
+// RUN: circt-opt -pass-pipeline='builtin.module(firrtl.circuit(firrtl-lower-signatures))' --allow-unregistered-dialect %s | FileCheck --check-prefixes=SIG %s
```

Comment:
There should probably be a few `SIG*:` checks in the file that actually test the signature lowering :slightly_smiling_face:

Reply:
Or maybe a separate `lower-signatures.mlir` file?

Reply:
yea, was adding some failure tests.  checks in that file are one the way.